server/src/main/java/io/druid/client/cache/MemcachedCache.java

```diff
@@ -473,9 +473,6 @@ public CacheStats getStats()
         return null;
       }
     }
-    catch (IOException e) {
```

Comment:
they're "related"-ish in that none of the ResourceHolder impls actually throw IOExceptions on close, and I didn't want to have IOException guards around the new calls to "close", so I thought it would be nice to remove the unnecessary "throws IOException". Removing that means none of the existing guards would compile so I had to take them out.